src/Mono.Android/Profiles/api-25.xml.in

```diff
@@ -17373,7 +17373,7 @@
         </parameter>
       </method>
       <method abstract="false" deprecated="not deprecated" final="false" name="writeNewStateDescription" native="false" return="void" static="false" synchronized="false" visibility="public">
-        <parameter name="fd" type="android.os.ParcelFileDescriptor">
+        <parameter name="p0" type="android.os.ParcelFileDescriptor">
```

Comment:
I'm assuming that we don't need to care about these "parameter name 'regressions'" because the `api-25.params.txt` has the correct parameter name?

Reply:
Yup, `api-25.params.txt` contains:

```
  class FileBackupHelperBase
    writeNewStateDescription(android.os.ParcelFileDescriptor fd)
```